lnrpc/invoicesrpc/addinvoice.go

```diff
@@ -397,6 +406,7 @@ func AddInvoice(ctx context.Context, cfg *AddInvoiceConfig,
 			Expiry:          payReq.Expiry(),
 			Value:           amtMSat,
 			PaymentPreimage: paymentPreimage,
+			PaymentAddr:     paymentAddr,
```

Comment:
The pay address isn't exposed on for example `ListInvoices`

Reply:
Features not either I noticed

Reply:
both are exposed in the original pr